config/locales/client.en.yml

```diff
@@ -592,22 +592,34 @@ en:
         pending:
           title: "Pending"
         approved:
+          title: "Approved"
+        approved_flag:
           title: "Flag approved"
+        approved_user:
+          title: "User approved"
+        approved_post:
+          title: "Post approved"
         rejected:
+          title: "Rejected"
+        rejected_flag:
           title: "Flag rejected"
+        rejected_user:
+          title: "User rejected"
+        rejected_post:
+          title: "Post rejected"
         ignored:
           title: "Flag ignored"
         deleted:
           title: "Topic or post deleted"
         reviewed:
-          title: "(all reviewed)"
+          title: "All reviewed"
         all:
-          title: "(everything)"
+          title: "Everything"
 
       context_question:
         is_this_post: "Is this post %{reviewable_human_score_types}?"
         delimiter: "or"
-        something_else_wrong: "Is there something else wrong with this post?"
+        something_else_wrong: "Is there something wrong with this post?"
```

Comment:
Yes, this was feedback from James that I agree with.

Reply:
I think it was this way because we display `Something else` as the flag reason, but the new translation makes more sense to me than this:

<img width="427" alt="Screen Shot 2023-03-09 at 12 07 47" src="https://user-images.githubusercontent.com/5025816/224066227-f65fdf5f-34c1-4369-b088-9136c74deff5.png">